README.rst

```diff
@@ -29,64 +31,138 @@ Contributing
 ============
 
 Contribute new tools by making a Pull Request on this repository.
-Please create a directory per contribution.
-(We may re-organize if it becomes obvious that doing so would 
+Please create a new directory per contribution.
+(We may re-organize if it becomes obvious that doing so would
 make it easier for users to find useful logic.)
 Please include a README file (README.rst or other form)
-describing any information about your tool, including the
-license it is released under.
+describing any information about your tool, *including the
```

Comment:
Maybe "If at all possible please use the MIT license" ?

Reply:
Hmm, line 30 had a change I didn't intend, s/if/this/.

Reply:
> Maybe "If at all possible please use the MIT license" ?

made that change.